apps/api/src/app/environments-v1/usecases/output-renderers/expand-email-editor-schema-command.ts

```diff
@@ -4,6 +4,6 @@ import { BaseCommand } from '@novu/application-generic';
 import { FullPayloadForRender } from './render-command';
 
 export class ExpandEmailEditorSchemaCommand extends BaseCommand {
-  body: string;
+  emailEditorJsonAsString: string;
```

Comment:
i think the type is enough to tell us it is a string, no? 

```suggestion
  emailEditorJson: string;
```